earthpy/plot.py

```diff
@@ -149,11 +163,22 @@ def plot_bands(
         axs_ravel = axs.ravel()
         for ax, i in zip(axs_ravel, range(total_layers)):
             band = i + 1
-            ax.imshow(es.bytescale(arr[i]), cmap=cmap)
+            if scale:
```

Comment:
yes!! i just noticed that as well and it's fixedc now :)

Reply:
for this one however --   i didn't do it because i need to somehow capture the arr at a specific dim. and i didn't want to overwrite the variable arr since there are mutliple dimensions here. Any thoughts on how to account for this?      
```
 if scale:
                arr_im = es.bytescale(arr[i])
            else:
                arr_im = arr[i]
```

Reply:
@lwasser isn't `arr_im` already being overwritten/reassigned for every iteration in this loop? If so, then it seems like the simpler `if` logic should still work unless I'm missing something.

Reply:
Also, the suggestion I made should not overwrite the variable `arr`, right? It would conditionally overwrite `arr_im`.

Reply:
ok i think i follow now. let me try this.